pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,502 @@
+        return 0
+
+    # Step 1: Fetch metadata from BigQuery (1 query for all file_ids)
+    file_ids = list(temp_files_map.keys())
+    metadata_query = f"""
+      SELECT
+        files_id as file_id,
+        filename,
+        contenttype
+      FROM `rj-nf-agent.poc_osinfo_ia.target_files`
+      WHERE files_id IN UNNEST(@file_ids)
+    """
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)